recipes-nodes/reth/init

```diff
@@ -58,10 +62,29 @@ start() {
 }
 
 stop() {
-	echo -n "Stopping $DESC: "
-	start-stop-daemon -K -x "$DAEMON" -p $PIDFILE
-	echo "$NAME."
+    echo "Stopping $NAME"
+    if [ -f $PIDFILE ]; then
+        PID=$(cat $PIDFILE)
+        # Find all child processes of the shell process
+        PIDS="$PID $(pgrep -P $PID)"
+
+        # Send SIGTERM to all processes
+        for pid in $PIDS; do
```

Comment:
This can also be done through `pkill -P $(cat $PIDFILE)` if pkill is available

Reply:
I used pkill before but for some cases it wasn't taking any effect. So I did it like this here and made it consistent across all init scripts